src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -37,6 +44,8 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
   session.execute(createTable)
   session.execute(createMetatdataTable)
   session.execute(createConfigTable)
+  for (tagId <- 1 to maxTagId)
```

Comment:
No, max 3, depends on what tags (the tag id 1, 2, 3) that user has configured.
I choose max 3 as a reasonable tradeoff between flexibility and overhead. I would be surprised if cassandra can't manage 3 such views.